modules/eforms/pages/roadmap/index.adoc

```diff
@@ -1,15 +1,21 @@
 = eForms SDK Roadmap
 
-link:https://docs.ted.europa.eu/eforms/latest/versioning.html[SDK versions] are released several times per year to improve the provided metadata or to implement regulatory changes. Releases are announced directly on GitHub, where you can watch for updates or follow the discussions. 
+SDK versions are released several times per year to improve the provided metadata and implement regulatory changes. Releases are announced in the link:https://github.com/OP-TED/eForms-SDK/discussions/categories/announcements[eForms developer forum on GitHub], where you can watch for updates or follow the discussions. 
 
-xref:active-versions/index.adoc[Active SDK versions] may also be patched, which currently applies the latest labels/translations and view-templates to previous versions. In exceptional cases, such as legal corrections, certain rule changes may also be backported in a patch as long as they never make the rules stricter.
-
-link:https://docs.ted.europa.eu/eforms-sdk-explorer/[The SDK Explorer] can help you with upgrading between versions. 
+This page provides an overview of the upcoming releases and the changes that are planned for them. As it is updated manually it may not always be up to date. The last revision to this page was made in March 2024.
 
 == eForms SDK 1.x.x
 
 Since the release of the eForms SDK 1.0.0 in August 2022, we have continued to release updates to the SDK that incrementally improve the eForms metadata that is included in it. These updates are all backwards compatible with the SDK 1.0.0 and therefore they can be used by applications that have already been adapted to the SDK 1.0.0. 
 
+=== SDK 1.12.0
+SDK 1.12.0, which will be released shortly after SDK 1.11.0 in March 2024, will not include any metadata updates but **will extend the conceptual model** by introducing **Business Entities**  in `fields.json`. 
```

Comment:
I'm not sure it will be in March, so we could leave it out, and just say "shortly after SDK 1.11.0".

Reply:
I removed the whole thing (actually commented it out so that we can use it when the time comes) because also Karl did not want to announce this before we get the dates right.